common/scripts/lib/_helpers.sh

```diff
@@ -303,6 +303,12 @@ __set_secret_key() {
   __process_success "Please enter the provided installer secret key."
   read response
   export SECRET_KEY="$response"
+
+  local get_ips=$(ip -f inet -4 addr | grep inet | grep -v 127.0.0.1 | awk '{print $2}' | awk -F "/" '{print $1}');
+
+  if [[ "$get_ips" ]]; then
```

Comment:
add a logger here in both if and else blocks to say whether ips are found or not. 
- if found, then something like `x ip addresses available for the host`
- if not, `no ip addresses avaiable`